addon/ng2/models/config.ts

```diff
@@ -27,9 +24,22 @@ function _findUp(name: string, from: string) {
 
 
 export class CliConfig {
-  constructor(private _config: CliConfigJson = CliConfig.fromProject()) {}
+  _config: any;
```

Comment:
Yes, we set variables with `set()` method and not with direct allocation as we don't use proxies now.